pkg/util/time.go

```diff
@@ -0,0 +1,7 @@
+package util
+
+import "time"
+
+func TimeMilliseconds(t time.Time) int64 {
+	return t.UnixNano() / int64(time.Millisecond/time.Nanosecond)
```

Comment:
`time.Nanosecond` is `1` so it's enough to do `/ int64(time.Millisecond)`.

Reply:
It's not something we want to rely on. It's an implementation detail and this is already going to be optimised by the compiler anyways.